build-tools/scripts/PrepareWindows.targets

```diff
@@ -5,28 +5,27 @@
     <_TopDir>$(MSBuildThisFileDirectory)..\..</_TopDir>
     <_NuGet>.nuget\NuGet.exe</_NuGet>
   </PropertyGroup>
-  <ItemGroup>
-    <_ConfigurationFile Include="Windows-Configuration.OperatingSystem.props">
-      <Destination>$(_TopDir)\Configuration.OperatingSystem.props</Destination>
-    </_ConfigurationFile>
-    <_ConfigurationFile Include="Configuration.Java.Interop.Override.props">
-      <Destination>$(_TopDir)\external\Java.Interop\Configuration.Override.props</Destination>
-    </_ConfigurationFile>
-  </ItemGroup>
   <UsingTask AssemblyFile="$(_TopDir)\bin\Build$(Configuration)\xa-prep-tasks.dll" TaskName="Xamarin.Android.BuildTools.PrepTasks.JdkInfo" />
+  <UsingTask AssemblyFile="$(_TopDir)\bin\Build$(Configuration)\xa-prep-tasks.dll" TaskName="Xamarin.Android.BuildTools.PrepTasks.ReplaceFileContents" />
   <Target Name="Prepare">
     <Exec Command="git submodule update --init --recursive" WorkingDirectory="$(_TopDir)" />
-    <Copy
-        SourceFiles="@(_ConfigurationFile)"
-        DestinationFiles="@(_ConfigurationFile->'%(Destination)')"
-        SkipUnchangedFiles="True"
-    />
     <MSBuild Projects="$(MSBuildThisFileDirectory)..\xa-prep-tasks\xa-prep-tasks.csproj" />
     <JdkInfo
         AndroidSdkPath="$(AndroidSdkPath)"
         AndroidNdkPath="$(AndroidNdkPath)"
         JavaSdkPath="$(JavaSdkDirectory)"
-        Output="$(_TopDir)\external\Java.Interop\bin\BuildDebug\JdkInfo.props"
+        Output ="$(_TopDir)\external\Java.Interop\bin\BuildDebug\JdkInfo.props">
+        <Output TaskParameter="JavaSdkDirectory" PropertyName="_JavaSdkDirectory" />
```

Comment:
This is indented too much; it should be 2 spaces from parent element column, not 4. (*Attributes* are 4 spaces.)